src/Identity/testassets/Identity.DefaultUI.WebSite/StartupBase.cs

```diff
@@ -77,9 +77,23 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseStaticFiles();
             app.UseCookiePolicy();
 
+            app.UseRouting();
+
             app.UseAuthentication();
 
-            app.UseMvc();
+            // This has to be disabled due to https://github.com/aspnet/AspNetCore/issues/8387
+            //
+            // UseAuthorization does not currently work with Razor pages, and it impacts
```

Comment:
eep

Reply:
yep. bugs happen every way.

This was the point where I decided that `UseMvc` needed to stop working with ER.